pkgs/development/python-modules/python-keycloak/default.nix

```
@@ -3,29 +3,36 @@
 , fetchFromGitHub
 , requests
 , python-jose
+, poetry
+, pytest
 , httmock
 , unittestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "python-keycloak";
-  version = "0.27.0";
+  version = "2.6.0";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "marcospereirampj";
     repo = "python-keycloak";
-    rev = version;
-    sha256 = "sha256-XCOfzzUs0K5/peprgpEXY2pX6wYOF7hg9ec1XPEYHCI=";
+    rev = "v${version}";
+    sha256 = "sha256-cuj0gJlZDkbJ2HRSMcQvO4nxpjw65CKGEpWCL5sucvg=";
   };
 
+  buildInputs = [
+    poetry
```

The used build system is [`poetry-core`](https://github.com/marcospereirampj/python-keycloak/blob/master/pyproject.toml#L76).

```suggestion
    poetry-core
```